### ISSUE TITLE
Indentation issues when moving lines

### ISSUE THREAD

**Issue by okapii** [godotengine/godot-vscode-plugin#130]
Is it possible that the Godot VS Code extension shares code with the python extension? It exhibits the same behaviour that has been found to be a bug for the python extension:

https://github.com/microsoft/vscode/issues/78788

**Comment by okapii** [comment]
This issue is still unresolved. Any chance that somebody might look into this at some point? The python plugin (see link in original post above) seems to have fixed this by now so maybe that solution would just have to be ported over?

**Comment by AntonBogun** [comment]
this is still a thing.

**Comment by Calinou** [comment]
Related to https://github.com/godotengine/godot-vscode-plugin/issues/337.

**Comment by Calinou** [comment]
Code sample for reproducing this courtesy of @froggestspirit:

> From https://github.com/microsoft/vscode-python/issues/6886, select the line "move_to_end", then keep pressing ALT+Down to move it to the end

```
move_to_end


class Foo:
    def __init__(self):
        pass

    def bar(self):
        pass


class Baz:
    pass
```

**Comment by brokenbeta** [comment]
Yep, was going to report this, would love a fix as it's making rearranging GDScript fairly painful -- pretty much any manipulation where the lines being moved cross over anything complicated, the move will change the indentation of the code behind and break it

**Comment by DaelonSuzuka** [comment]
I don't believe this is fixable. The indentation isn't automatically changed in python, either, and the language configuration is controlled by Microsoft themselves. I suspect that VSCode doesn't support/expose hooks that allow you to define indent/dedent on line movement.

**Comment by DaelonSuzuka** [comment]
The indentation changes in your gif in GDScript look correct, actually. I don't understand what your complaint is.

I also don't understand why it's happening, because this is what I get:

![Code_nruVRN0uSH](https://github.com/godotengine/godot-vscode-plugin/assets/18042232/855db2d0-a60d-45ad-a789-b3d58c004630)

This is with zero extensions installed except this one, and `editor.autoIndent` set to `true`.

**Comment by HolonProduction** [comment]
I can't reproduce this on the current version as well. Let's close it until someone actually has reproduction steps for it.
